source/include/FreeRTOS_DNS_Globals.h

```diff
@@ -331,4 +321,14 @@
 
     #endif /* ( ipconfigUSE_LLMNR == 1 ) || ( ipconfigUSE_NBNS == 1 ) */
 #endif /* ipconfigUSE_DNS */
+
+/* Keeping this outside of ipconfigUSE_DNS flag as these are used inside IPv4 UDP code */
+#define ipMDNS_PORT              5353U       /* Standard mDNS port. */
```

Comment:
Why is the code moved here from top?

Reply:
These macros are used outside of ipconfigUSE_DNS flag as these are used inside IPv4 UDP code. I have added a comment above.